rfcs/text/i18n.md

```diff
@@ -0,0 +1,270 @@
+- Start Date: 2020-06-14
+
+# Summary
+
+Introduce first class support for internationalisation and localisation within `carbon-react`.
+
+# Basic example
+
+The proposal is to introduce a new component `I18nProvider` that allows our consumers to provide a 
+[`react-i18next`](https://github.com/i18next/react-i18next) compatible<sup>[1] [2]</sup> `t` function.
+
+We will need to check each instance that we use our current `i18n-js` implementation and ensure that the
```

Comment:
I think it will look better
```suggestion
We will need to check each instance that we use our current `i18n-js` implementation and ensure that the:
```

Reply:
I also don't think this sentence makes sense looking at it - maybe "We will need to check each instance where our current `i18n-js` implementation is used" ?

Reply:
Changed to
> We will need to check each usage of our current `i18n-js` implementation to ensure the: